doajtest/helpers.py

```diff
@@ -3,43 +3,87 @@
 from unittest import TestCase
 from portality import core, dao
 from portality.app import app
-from doajtest.bootstrap import prepare_for_test
+from portality.tasks.redis_huey import main_queue, long_running
 import dictdiffer
 from datetime import datetime
 from glob import glob
 import os, csv, shutil
 from portality.lib import paths
+import functools
+import time
 
-prepare_for_test()
 
+def patch_config(inst, properties):
```

Comment:
this is nifty. However, if a test has specific overrides you'd have to make sure you don't assign to the same `originals` or those will be lost

Reply:
Yes, each test will have to store its own changes in its own member variable.  It'll break if you don't, I think, so should be easy to spot.